### PR TITLE
[4.0] Toolbar spacing

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for  #32197 requires npm i

### Before
image

### After
image

Thanks @dgrammatiko

### CODE REVIEW

File: administrator/templates/atum/scss/blocks/_toolbar.scss [hunk 1]
```diff
@@ -20,6 +20,11 @@
     box-shadow: none;
   }
 
+  .btn-group
+    > joomla-toolbar-button {
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_toolbar.scss]:
```suggestion
  .btn-group > joomla-toolbar-button {
```

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 6ede0f63482d046a55b82bebbf5316c21489af86<br /><br />With Prebuilt package.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32209">issues.joomla.org/tracker/joomla-cms/32209</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 6ede0f63482d046a55b82bebbf5316c21489af86<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32209">issues.joomla.org/tracker/joomla-cms/32209</a>.</sub>

**Comment by infograf768** [comment]
Please follow @Quy suggestion
https://github.com/joomla/joomla-cms/pull/32209#discussion_r567282038

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32209">issues.joomla.org/tracker/joomla-cms/32209</a>.</sub>

**Comment by infograf768** [comment]
merged as it does not change existing tests.

tks

**Comment by brianteeman** [comment]
thanks

**Pull request merged**